akka-docs-new/src/main/paradox/guide/actors-intro.md

```diff
@@ -0,0 +1,259 @@
+# What problems does the actor model solve? 
+
+Akka uses the actor model to overcome the limitations of traditional object-oriented programming models and meet the 
+unique challenges of highly distributed systems. To fully understand why the actor model is necessary, it helps to 
+identify mismatches between traditional approaches to programming and the realities of concurrent and distributed 
+computing. 
```

Comment:
I don't like using links because then they need to be separate pages, which end up in the TOC, polluting it anyway. Originally I had a sentence about the section being optional, it was removed by Ruth. I am fine with anything just decide what you want (or you can change it however you like).